File: Linguaggio_Alto_Livello/Java/Hard/SimpleContactBook.java
import java.util.LinkedHashMap;
import java.util.Map;

public class SimpleContactBook {
    public static void main(String[] args) {
        Map<String, String> contacts = new LinkedHashMap<>();
        contacts.put("Luca", "333-1111111");
        contacts.put("Marta", "333-2222222");
        contacts.put("Paolo", "333-3333333");

        System.out.println("Rubrica:");
        for (Map.Entry<String, String> entry : contacts.entrySet()) {
            System.out.println(entry.getKey() + ": " + entry.getValue());
        }
    }
}

File: Linguaggio_Medio_Livello/C/Hard/simple_contact_book.c
#include <stdio.h>

struct Contact {
    char name[20];
    char phone[15];
};

int main(void) {
    struct Contact contacts[] = {
        {"Luca", "333-1111111"},
        {"Marta", "333-2222222"},
        {"Paolo", "333-3333333"}
    };

    int size = sizeof(contacts) / sizeof(contacts[0]);

    printf("Rubrica:\n");
    for (int i = 0; i < size; i++) {
        printf("%s: %s\n", contacts[i].name, contacts[i].phone);
    }

    return 0;
}
